selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -607,4 +607,31 @@ public String getModel() {
   public String getAPITargetType() {
     return apiTargetType;
   }
+
+  public void unlockScreen() throws AndroidDeviceException {
+    // Get phone's android version and whether the screen is off.
+    // Different ways to detect if the screen is off depending on the version.
+    CommandLine commandVersion = adbCommand("shell", "getprop", "ro.build.version.release");
+    String version = executeCommandQuietly(commandVersion);
+    CommandLine commandScreen = adbCommand("shell", "dumpsys", "power");
+    String screen = executeCommandQuietly(commandScreen);
+
+    // Kitkat
+    if (version.matches("^4\\.4(\\..*)?")) {
```

Comment:
i see you've only handled KitKat and Lollipop (likely cause that's all you have access to and I don't have access to older either right now)... but selendroid attempts to support back to Gingerbread (2.3.3 / API level 10). Do we have any idea what it may be there and the versions inbetween? If we can't tell, let's just try to have it do the same thing the KitKat one does here and maybe others with devices on the old versions can either verify or fix it later :)